index.js

```diff
@@ -189,16 +246,22 @@ function getGermanCars(/* code here */) {
  * }
  * 
  * const addFive = function(num) {
-*    return num + 5
+ *    return num + 5
  * }
  *
  * const argTimesTwo = function (num) {
  *   return num * 2
  * }
-*/
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+ */
+const sum = (a, b) => {
+    return a + b;
+};
+const addFive = num => {
+    return num + 5;
+};
+const argTimesTwo = num => {
+    return num * 2;
+};
 
```

Comment:
for lines 255-264: You do not need the return or curly braces.  With arrow functions they automatically include the `return` keyword so you can delete that and the curly braces.  For example line 256 should look like this ` const sum = ( a,b ) => a + b;`

Reply:
It was an oversight. Thank you!